harness/determined/common/api/authentication.py

```diff
@@ -304,8 +309,6 @@ def drop_user(self, username: str) -> None:
             tokens = substore.setdefault("tokens", {})
             if username in tokens:
                 del tokens[username]
-            if substore.get("active_user") == username:
```

Comment:
I don't think there's a test in the repo that specifically guards against regression on this bug. If you find writing one natural and easy, please! But 100% up to you whether you think it's worth your development time.

Reply:
I don't think it's worth it, honestly.

Reply:
Works for me!